src/components/templates/DataTable/styles.module.scss

```diff
@@ -0,0 +1,80 @@
+@import 'styles/variables';
+@import 'styles/mixins';
+
+.title {
+  margin: spacing(6) spacing(4);
+}
+
+.tableWrapper {
+  display: grid;
+  height: 100%;
+  overflow-x: auto;
+  white-space: nowrap;
+
+  .dataTable {
+    width: 100%;
+    text-align: left;
+    margin-bottom: 0;
+    display: table;
+    @include bodyTextStyle;
+  }
+
+  .l th,
+  td {
+    padding: 12px 14px;
```

Comment:
**[blocker]**

mentioned above as well, but a lot of values in this file that should use spacing